beacon-chain/entrypoint.sh

```diff
@@ -22,16 +22,11 @@ case $_DAPPNODE_GLOBAL_EXECUTION_CLIENT_PRATER in
     ;;
 esac
 
-# MEVBOOST: https://github.com/ChainSafe/lodestar/blob/unstable/docs/usage/mev-integration.md
+# MEV-Boost: https://chainsafe.github.io/lodestar/usage/mev-integration/
 if [ -n "$_DAPPNODE_GLOBAL_MEVBOOST_PRATER" ] && [ "$_DAPPNODE_GLOBAL_MEVBOOST_PRATER" == "true" ]; then
-    echo "MEVBOOST is enabled"
+    echo "MEV-Boost is enabled"
     MEVBOOST_URL="http://mev-boost.mev-boost-goerli.dappnode:18550"
-    if curl --retry 5 --retry-delay 5 --retry-all-errors "${MEVBOOST_URL}"; then
```

Comment:
all other packages only check if mev-boost is reachable in validator entrypoint script, I would assume a user would get two notifications if we have this check also in the beacon-chain entrypoint script

Reply:
good catch however the outcome would be the same, it would only give 2 notifications when MEV-Boost is enabled but not connected to either service of the CL.  none if it is working. 
Thinking further though, all CLs need MEV-Boost params entered in both services (sans Nimbus the monoservice), and need both services to connect to MEV-Boost in order to work properly, so i think we should keep the check here and add checks to the beacon chains of lighthouse, Prysm, and Teku, otherwise it may not connect on the beaconchain service but the user would get no notification if this failure to connect.   @pablomendezroyo What do you think?

Reply:
@alexpeterson91 I moved the mev-boost health check to the beacon node, this makes much more sense as the validator doesn't even communicate with the builder directly, only the beacon node does.